packages/create-pantheon-decoupled-kit/src/templates/next-drupal/.env.example

```diff
@@ -1,14 +1,23 @@
-# Copy as .env.development.local to override envars for local development
+# The URL of your Drupal site
 BACKEND_URL=http://my-backend-site.pantheonsite.io
-FRONTEND_URL=http://my-frontend-site.pantheonsite.io
-IMAGE_DOMAIN=my-frontend-site.pantheonsite.io
-PREVIEW_SECRET=mysecret
+
+# The domain where your images are hosted. By default, we will set the domain of the BACKEND_URL
+# to the value of IMAGE_DOMAIN. Set this if your images are hosted somewhere other than your WordPress instance
```

Comment:
Some instances of WordPress here that need to be changed to Drupal.

Reply:
Oops!